### DISCUSSION TITLE
ödevi teslim alamıyorum

### DISCUSSION THREAD

**Discussion by Mlsoznar** [19/classroom#20] [Anonim]
Tüm kurulumları eksiksiz yaptım.Attığınız ödev kısmında en son readme.md  visual studio code a yönlendirme kısmında visual studio code a yönlendiriliyor fakat readme.md dosyası açılmıyor yani visual studio code tepkisiz kalıyor. Ne yapmam gerekir.

**Comment by roktas** [comment]
#16 iletisini incele.  @Mlsoznar yalnız bak bu mesajı 1 dk önce atmışsın, halbuki #16 4 saat önce cevaplanarak sonlanmış.  Demem o ki sorun hakkında gözü kapalı mesaj yazmak yerine önce mevcut mesajları incele ve oradaki çözümleri bir dene.

**Reply by Mlsoznar** [reply]
hocam o çözümü de inceledim. Ctrl+Shift+P yaptıktan sonra çıkan yere " git klonla" yazıyorum ama çıkmıyor.O yüzden çözüm bulamadım.

**Reply by AhmetOzgul** [reply]
@Mlsoznar "git klonla" şeklinde değil de "Git: Clone" olarak aratmayı deneyebilirsin. Ben de Türkçe aradığımda çıkmamıştı bu şekilde yapabildim.
![Screenshot_30](https://user-images.githubusercontent.com/92336668/158067584-5645420c-4413-4ba4-b3b1-98b5ac7eec8f.png)

**Reply by roktas** [reply]
@Mlsoznar Yardımcı olmak amacıyla yazılan cevaplara gecikmeden dönüş yapın lütfen, "oldu/olmadı teşekkürler" şeklinde.

**Comment by roktas** [comment]
Kurulumunda bir sorun var muhtemelen, bu ödevden bağımsız.  Classroom kurulumuyla ilgili hataların raporlandığı yere (bu araştırıp öğren) taşı bu sorunu.  Raporlama yaparken de gerekli tüm bilgileri ve ekran görüntülerini sun.

Ödeve gelince, Classroom ortamı yerine alternatif yötemler kullan, duyuruda buna değindim.

**Comment by roktas** [comment]
> hocam o çözümü de inceledim. Ctrl+Shift+P yaptıktan sonra çıkan yere " git klonla" yazıyorum ama çıkmıyor.O yüzden çözüm bulamadım.

Soruyu sorarken bunu da yazmalıydın, "falan çözümü inceledim.  Ctrl+Shift+P yaptıktan sonra ..." gibi.  Öteki türlü yeterli ipucu vermeden yardım almanız zor.

Bakın bunları böyle uzun uzun yazmamın sebebi sizleri fırçalamak değil.  Derdini doğru anlatmak eğitimin ve bu ödevlerin bir parçası.  Meslek hayatınızda çok karşılaşacaksınız bu dediğimle.  Gelecekte bu sözlerimi daha iyi anlarsınız.

**Comment by Mlsoznar** [comment]
Oldu teşekkür ederim

14 Mar 2022 Pzt 01:32 tarihinde Recai Oktaş ***@***.***> şunu
yazdı:
